packages/gatsby-source-contentful/src/fetch.js

```diff
@@ -291,7 +291,7 @@ export async function fetchContent({ syncToken, pluginConfig, reporter }) {
           resolveLinks: false,
         }
         const query = syncToken
-          ? { nextSyncToken: syncToken, ...basicSyncConfig }
+          ? { nextSyncToken: syncToken }
```

Comment:
The Contentful API was failing for me, saying that it's not allowed to pass extra values along with nextSyncToken. Not sure why it hasn't failed for existing sites.

Reply:
Probably a question for @axe312ger . That is quite odd we haven't heard reports of it. Did the error say which extra values weren't allowed?

Reply:
The error message said that no other values should be sent when including a sync token.